flocker/control/_model.py

```diff
@@ -90,9 +90,7 @@ class IRestartPolicy(Interface):
 
 
 @implementer(IRestartPolicy)
-@attributes([], apply_immutable=True,
-            # https://github.com/hynek/characteristic/pull/22
-            apply_with_init=False)
+@attributes([Attribute("name", default_value=u"never")], apply_immutable=True)
```

Comment:
If you ever *do* need to do something like this, class attribute would be better rather than instance attribute:

```
class MyClass:
    name = "blah"
```